_pages/get_involved.md

```diff
@@ -73,14 +73,16 @@ will have to wait for BIDS 2.0 are listed
 | [BEP032](https://bids.neuroimaging.io/bep032){:target:"_blank"} | Animal electrophysiology                                                                                   | Sylvain Takerkart and Julia Sprenger                             | New BEP, collecting comments and community feedback which may be submitted [here](https://github.com/INCF/neuroscience-data-structure/issues), directly to [Sylvain](mailto:sylvain.takerkart@univ-amu.fr) & [Julia](mailto:julia.sprenger@univ-amu.fr), or [here](https://www.incf.org/sig/incf-working-group-standardized-data){:target:"_blank"} during the ~monthly meeting. Example datasets [here](https://gin.g-node.org/NeuralEnsemble/BEP032-examples){:target:"_blank"}. Supporting early stage interaction with: [ProbeInterface & SpikeInterface](https://github.com/SpikeInterface){:target:"_blank"}, [AnDO](https://github.com/INT-NIT/AnDO){:target:"_blank"}, Interoperability with NWB and NIX data/metadata formats.                                                                                                                    | None.                                           |
 | [BEP033](https://bids.neuroimaging.io/bep033){:target:"_blank"} | Advanced Diffusion Weighted Imaging (aDWI)                                                                 | James Gholam, Leandro Beltrachini and Filip Szczepankiewicz      | New BEP, seeking contributors and collecting community feedback relating to: priority sequences to support, best supported binary structured formats (e.g. CBOR? HDF5? MsgPack?) Comments may be submitted directly on the document. Generating example datasets [here](https://github.com/JAgho/MISP_plot/tree/main){:target:"_blank"} and Determining best practice w/ vendors to record data in-sequence | None.                                           |
+| [BEP034](https://bids.neuroimaging.io/bep034){:target:"_blank"} | Computational modeling                                                                                     | Michael Schirner and Petra Ritter                                |  | None.                                           |
+| [BEP035](https://bids.neuroimaging.io/bep035){:target:"_blank"} | Extended derivatives structure for individual-level meta-analyses based on non-BIDS compliant datasets     | Giuseppe Gallitto and Tamas Spisak                               |  | None.                                           |
```

Comment:
New BEP, collecting comments and community feedback, especially on handling "conditions-of-interests". Collaborators with mega-analysis experience or with actual example use-cases (planned/ongoing/finished mega analyses) welcome!

Reply:
Thanks!